### PR TITLE
Refactor meeting information to use calendar as source of truth

### PR DESCRIPTION
**Pull request by choldgraf**
This attempts to refactor our team meetings information in order to be easier to find and more clearly distinguish a "source of truth" for meetings.

Previously, we documented both the _categories of meetings_, as well as _instances of each type of meeting_ with links to Google calendar, notes, etc. This was confusing because it meant that there was information about our meetings both in the team compass, the calendar, and in our notes.

This PR tries to improve this with a few steps:

- Makes the **team events calendar the source of truth** for "when and what are our specific team meetings".
- Makes the **notes for each meeting the source of truth for purpose and content**. These are in tables at the top of each rolling notes document rather than being defined in the team compass directly.
- Creates an org-wide page that describes our **meeting policy and categories of meetings**, but not specific meeting details.
- Generally tries to simplify language to be simple and memorable, and to let meeting facilitators decide how to interpret these guidelines for their own team practices.
- Moves the old engineering team notes into a sub-folder of `engineering` rather than being org-wide.

References:

- closes https://github.com/2i2c-org/team-compass/issues/564
- related to https://github.com/2i2c-org/team-compass/issues/496
- touches on https://github.com/2i2c-org/team-compass/issues/517 but doesn't define any new kind of team meeting

### CODE REVIEW

File: reference/calendar.md [hunk 1]
```diff
@@ -1,31 +1,27 @@
...
+<iframe src="https://calendar.google.com/calendar/embed?src=c_4hjjouojd8psql9i1a8nd1uff4%40group.calendar.google.com" style="border: 0; width:100%"  height="600" frameborder="0" scrolling="no"></iframe>
 
 ```
 
 (team-roles-calendar)=
 ## Team Roles Calendar
 
 We use a Team Roles calendar to keep track of when each team member is serving in a given role.
-You can find [the Team Roles calendar here](https://calendar.google.com/calendar/embed?src=c_nq8hl7qsm484g1p7mfkm29jpo8%40group.calendar.google.com&ctz=America%2FLos_Angeles).
+You can find [the Team Roles calendar here](https://calendar.google.com/calendar/u/1?cid=Y19ucThobDdxc200ODRnMXA3bWZrbTI5anBvOEBncm91cC5jYWxlbmRhci5nb29nbGUuY29t).
```
Comment (sgibson91) [inline reference/calendar.md:20]: Another `/u/1` link here

Comment (choldgraf) [reply reference/calendar.md:20]: gah this drives me nuts - thanks for finding

**Comment by choldgraf** [comment]
OK I've fixed up the calendar links (and found a few others that were messed up as well). Thanks for the review @sgibson91 !

File: operations/meetings.md [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+
+- Discussing new ideas and proposals
+
+(coordination:team-syncs)=
+### Asynchronous team syncs
+
+A lightweight way to share what we've been up to and signal-boost items for other team members to pay attention to.
+These are managed by [an automated Geekbot questionnaire](https://geekbot.com/) in [our team slack](communication:slack).
+
+**What these meetings for**:
```
Comment (damianavila) [inline operations/meetings.md:107]: No actual meetings 😉

**Pull request merged**